ariadne/cardano/src/Ariadne/Wallet/Backend/Restore.hs

```diff
@@ -91,10 +91,7 @@ restoreFromKeyFile ::
     -> WalletRestoreType
     -> IO ()
 restoreFromKeyFile pwl face runCardanoMode mbWalletName path rType = do
-    keyFile <- BS.readFile path
-    us <- case decodeFull' keyFile of
-      Left e   -> throwM $ SecretsDecodingError path e
```

Comment:
AFAIU, after this deletion `SecretsDecodingError` (defined in this file) is not used anymore, so it should be deleted.